controllers/openstackcluster_controller.go

```diff
@@ -456,3 +481,9 @@ func (r *OpenStackClusterReconciler) SetupWithManager(ctx context.Context, mgr c
 		WithEventFilter(predicates.ResourceNotPausedAndHasFilterLabel(ctrl.LoggerFrom(ctx), r.WatchFilterValue)).
 		Complete(r)
 }
+
+func handleUpdateOSCError(openstackCluster *infrav1.OpenStackCluster, message error) {
+	err := capierrors.UpdateMachineError
```

Comment:
Since this is openStackCluster, not openStackMachine, we should use [ClusterStatusError](https://github.com/kubernetes-sigs/cluster-api/blob/master/errors/consts.go#L83).

Reply:
Thanks. Fixed.